docs/README.md

```diff
@@ -1,6 +1,43 @@
 # auth-service/Docs
 
 Documentation for the auth service
+## Database
+
+The following Figure depicts the Entity Relationship Diagram of the auth-service's schema:
+
+<p align="center">
+  <img src="http://www.plantuml.com/plantuml/proxy?cache=no&src=https://raw.githubusercontent.com/spikerheado1234/auth-service/master/docs/ErDiagram.puml">
+</p>
+
+The following table depicts the types of each attribute in the `Consent` table along with the constraints:
+
+|Attribute| Represented Internal Type | Constraints| Description |
+|:---------:|:---------------------------:|:------------:|:---------------------------:|
+|`id`|`VARCHAR(36)`|Unique - Primary Key|Alphanumeric consent ID|
+|`initiator_id`|`VARCHAR(32)`|Non-Null|Alphanumeric PISP ID|
+|`participant_id`|`VARCHAR(32)`|Non-Null|Alphanumeric DFSP ID|
+|`created_at`|`TIMESTAMP`|Non-Null|Timestamp of consent creation|
+|`credential_id`|`CHAR(256)`|Nullable|Unsigned numeric ID - upto 2^32 records|
```

Comment:
https://codelabs.developers.google.com/codelabs/fido2-for-android/#5 also shows a string being used.

```suggestion
|`credential_id`|`VARCHAR(256)`|Nullable|Unsigned numeric ID - upto 2^32 records|
```